cm-mlops/automation/script/module.py

```diff
@@ -3229,7 +3296,16 @@ def prepare_and_run_script_with_postprocessing(i, postprocess="postprocess"):
     script_automation = i['self']
 
     # Preapre script name
-    script_name = 'run' if i.get('script_name','')=='' else i['script_name']
+    check_if_run_script_exists = False
+    script_name = i.get('script_name','').strip()
+    if script_name == '':
+        script_name = meta.get('script_name','').strip()
+        check_if_run_script_exists = True
```

Comment:
We need "if script_name != '' " to turn on the check flag right?

Reply:
Yes, I am fixing it now ;) ... Thanks.